MaxText/input_pipeline/input_pipeline_interface.py

```diff
@@ -31,10 +31,10 @@
 import multihost_dataloading
 
 
-def get_tokenizer(tokenizer_path, add_bos=True, add_eos=True):
+def get_tokenizer(tokenizer_path, add_bos, add_eos):
```

Comment:
why did we need to remove the default values of `add_bos/add_eos`?

Reply:
There were multiple places that had default values set for add_bos/add_eos. I thought of simplifying it and keeping the default value at one places [L224](https://github.com/google/maxtext/blob/mohit/llama3/MaxText/input_pipeline/input_pipeline_interface.py#L224). Now those same default values get funnel through all the functions which need them

Reply:
SG - thanks Mohit